source/authentication/use-firebase-auth.txt

```diff
@@ -0,0 +1,189 @@
+.. _custom-jwt-authentication-firebase:
+
+========================================
+Use Firebase Authentication (Custom JWT)
```

Comment:
This is the only one of the Auth pages that uses "Use" in the title. It also does not match the TOC, so I'd suggest removing it here.

Reply:
I was thinking of this as a pattern we could use for integration guides, i.e. "Use <Foo>". I'll remove for now and we can revisit once we have more guides.